src/app/js/admin/upload/Upload.js

```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import RaisedButton from 'material-ui/RaisedButton';
+import ArchiveIcon from 'material-ui/svg-icons/content/archive';
+
+import { uploadFile } from './';
+
+export const UploadComponent = ({ onFileLoad }) => (
+    <div
+        style={{
```

Comment:
put styles in an object with keys for each "classes". Make the react code easier to read